pkgs/by-name/ha/haredo/package.nix

```diff
@@ -41,20 +44,21 @@ stdenv.mkDerivation (finalAttrs: {
 
   enableParallelChecking = true;
 
-  doCheck = true;
+  doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
 
   dontConfigure = true;
 
   preBuild = ''
-    HARECACHE="$(mktemp -d --tmpdir harecache.XXXXXXXX)"
+    HARECACHE="$(mktemp -d)"
```

Comment:
Any reason this particular change is included in this PR?

Reply:
There was no reason for it to be there in the first place: hare just needs a directory for its cache. By using `mktemp -d`, we make it clear that it's just a directory, nothing else.